src/MensaWidget/MWFrameTest.class.st

```diff
@@ -91,6 +91,22 @@ MWFrameTest >> testOpened [
 		assert: self widgetFrame isCollapsed = false
 ]
 
+{
+	#category : #tests,
+	#'squeak_changestamp' : 'Max 6/17/2024 11:59'
+}
+MWFrameTest >> testRegister [
+	
+	| occurrence |
+	
+	occurrence := 0.
+	TheWorldMenu registry do: [:each | each first = 'Mensa Widget' ifTrue: [occurrence := occurrence + 1]].
```

Comment:
between here we should have a register command since if it has not been called the test will fail and doing it multiple time does not change the behaviour. And then we will probably have to take care of the opened morph. As a solution the set up could be registy and then it might take care of itself.